features/swap/hooks/useSwapFee.ts

```diff
@@ -0,0 +1,61 @@
+import { NETWORK_FEE } from 'util/constants'
+import { useSwapInfo } from '../../../hooks/useSwapInfo'
+import { useTokenToTokenPrice } from './'
+
+export const useSwapFee = ({ tokenA, tokenB }) => {
+  const [tokenToTokenPrice] = useTokenToTokenPrice({
+    tokenASymbol: tokenA.tokenSymbol,
+    tokenBSymbol: tokenB.tokenSymbol,
+    tokenAmount: 1,
+  })
+
+  // pool_id for direct token swap pools OR inputPool for passthrough swap
+  let poolId1: string = ''
+  // pool_id for outputPool if passthrough swap
+  let poolId2: string = ''
+  if (tokenToTokenPrice.poolForDirectTokenAToTokenBSwap) {
```

Comment:
*NIT* This chain of if statements leads to me think the data structure returned by `useTokenToTokenPrice` and should probable be refactored in the future. I think the code here is good though given what we are working with.

Reply:
agreed! i think it would be nice if `useTokenToTokenPrice` returned a single array of pools ordered by max # of tokens user would receive, in the case of passthrough pools that would be an array nested inside the pools array. but yeah, i think we should look into it on another PR